MaxMind.Db.Benchmark/project.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "version": "2.0.0-*",
+  "description": "Benchmark project to validate .NET reader for the MaxMind DB file format",
+  "packOptions": {
+    "authors": [ "MaxMind, Inc." ],
+    "tags": [ "" ],
+    "projectUrl": "https://github.com/maxmind/MaxMind-DB-Reader-dotnet",
+    "licenseUrl": "https://github.com/maxmind/MaxMind-DB-Reader-dotnet/blob/v2.0.0/LICENSE",
+    "copyright": "Copyright © 2013-2016"
+  },
+
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "MaxMind.Db": "2.0.0-*"
+  },
+  "frameworks": {
+    "net451": { },
```

Comment:
Is there any reason to prefer `net451` over `net45`, which we targeted previously? I ask primarily because Mono users seem to have quite a few problems with `net451` and `net452`.